go/engine/login.go

```diff
@@ -70,6 +70,13 @@ func (e *Login) Run(ctx *Context) error {
 	if err == nil {
 		// login successful
 		e.G().Log.Debug("LoginCurrentDevice.Run() was successful")
+
+		// Unlock the passphrase stream here so the user won't have to
+		// separately
+		if _, err2 := e.G().LoginState().GetPassphraseStream(ctx.SecretUI); err2 != nil {
+			e.G().Log.Debug("Logged in, not handling passphrase stream error %s (err type: %T)", err2, err2)
```

Comment:
Do we want to ignore an error here?  Presumably an error is an invalid passphrase?

It seems if we ignore this error, then the user can be in a state without a passphrase stream cache.